code/game/objects/items/inquisition_relics.dm

```diff
@@ -952,11 +953,15 @@
 		C.visible_message(span_danger("[user] [pick("garrotes", "asphyxiates")] [C]!"), \
 		span_userdanger("[user] [pick("garrotes", "asphyxiates")] me!"), span_hear("I hear the sickening sound of cordage!"), COMBAT_MESSAGE_RANGE, user)
 		to_chat(user, span_danger("I [pick("garrote", "asphyxiate")] [C]!"))
-		user.changeNext_move(CLICK_CD_RESIST)	//Stops spam for choking.
+		var/pain = /obj/item/inqarticles/garrote
+		if(!pain)
+			user.changeNext_move(assassin_choke)	//Stops spam for choking.
+		else
+			user.changeNext_move(CLICK_CD_RESIST) //HAHAHAHA I WIN, I GET TO SLEEP NOW(to future coders, disregard this comment, it is the ramblings of a insomniac, no maintainers,Im not removing this.)
```

Comment:
Remove all of this and replace it with `user.changeNext_move(choke_change_next)` change `choke_change_next` for both garrotes as a replacement of `assassin_choke` and use SECONDS